pipelines/common/treatment/default_treatment/tasks.py

```
@@ -278,14 +281,19 @@ def task_dbt_selector_test_notify_discord(
     Args:
         contexts (list[DBTSelectorMaterializationContext]): Contexto de materialização.
     """
     for context in contexts:
-        context.selector.set_redis_materialized_datetime(
-            env=context.env, timestamp=context.datetime_end
-        )
+        try:
+            context.selector.set_redis_materialized_datetime(
+                env=context.env, timestamp=context.datetime_end
+            )
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)